Dockerfile

```diff
@@ -27,7 +28,8 @@ COPY --from=devkit /usr/local/bin/goss /usr/local/bin/
 COPY --from=devkit /usr/local/bin/mindthegap /usr/local/bin/
 COPY --from=devkit /usr/local/bin/packer /usr/local/bin/
 COPY --from=devkit /usr/local/bin/packer-provisioner-goss /usr/local/bin/
-COPY --from=devkit /root/.config/packer/plugins/ /root/.config/packer/plugins/
```

Comment:
when running kib wrapper as non-root user, the packer could not find and load the plugins and defaults to in-built version of the plugins. so moving the plugin to `/opt/packer/plugins` on the container so that packer can find correct version of the plugins.